content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md

```diff
@@ -19,6 +19,15 @@ package repositories (`pkgs.k8s.io`). If that's not the case, it's strongly
 recommended to migrate to the community-owned package repositories as described
 in the [official announcement](/blog/2023/08/15/pkgs-k8s-io-introduction/).
 
+{{< note >}}
+The legacy package repositories (`apt.kubernetes.io` and `yum.kubernetes.io`)
+have been frozen starting from September 13, 2023. To install Kubernetes versions
+released **after** September 13, 2013, you need to migrate to the community-owned
+package repositories. Please read our
+[deprecation and freezing announcement](/blog/2023/08/31/legacy-package-repository-deprecation/)
+for more details.
+{{< /note >}}
```

Comment:
Given that we have the same message across different documents, should we consider making this a shortcode? It might make tasks like translation easier.

Reply:
Might be more effort than you're hoping for though @xmudrii